index.html

```diff
@@ -37,10 +37,10 @@
 			Spare Parts: <progress id="spareParts" value="100" max="100"></progress>
 		</div>
 		<div>
-			<span id="invulnerabilityAvailable" style="visibility: visible;">
+			<span id="invulnerabilityAvailable" style="visibility: hidden;">

```

Comment:
Starts as hidden, only shows up when player gets the powerup (which happens when initialize game occurs)